### PR TITLE
Feature/pprof

### PR DESCRIPTION
**Pull request by johnabass**

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1 +1 @@
-
+- add pprof support [#50](https://github.com/xmidt-org/themis/pull/50)
```
Comment (joe94) [inline release_notes.md:1]: We can delete this file as we're no longer using it. We should move this entry to changelog.md

Comment (johnabass) [reply release_notes.md:1]: Is there anywhere this process is documented?

Comment (joe94) [reply release_notes.md:1]: I don't think we have it documented yet (I might be wrong, @kristinaspring, thoughts?). We've just been enforcing it in PRs.
But anyways, if it's not there, should it live in the "Pull Requests" section of the Contributing.md file?

Comment (johnabass) [reply release_notes.md]: Yeah.  We might want to file an issue in the appropriate repo, if the docs don't exist yet.

**Pull request merged**